### ISSUE TITLE
APRRES - to be released public project

### ISSUE THREAD

**Issue by xssfox** [aprsorg/aprs-deviceid#156]
```yml
 - tocall: APRRES
   vendor: xssfox
   model: APRS-RepeaterRescue
   class: network
   os: embedded
   contact: repeater-rescue@michaela.lgbt
   features:
      - messaging
```

**Comment by ldeffenb** [comment]
Is this related at all to the "Rescue Repeater" at https://www.beagleboard.org/projects/extending-communications-drone-based-radio-repeater

Extending Communications – Drone-based Radio Repeater
The "Rescue Repeater" is a drone-based radio repeater solution to allow communications to be rapidly restored or extended over the horizon.

**Comment by xssfox** [comment]
Not related

**Comment by hessu** [comment]
Hi,

There's no "network" class defined. I'm guessing this is a service or bot of some sort which sends and/or receives APRS messages? So I'd put it in the "service" class, is that alright?

**Comment by xssfox** [comment]
Service works for me. I got network from here

https://github.com/aprsorg/aprs-deviceid/blob/main/ALLOCATING.md#applicationdevice-classes

**Comment by hessu** [comment]
Ah right, they were missing in tocalls.yaml while they were defined in ALLOCATING.md. Fixed in #158 .

I'll make the allocation with "network" class which is for hardware appliances.
